### PR TITLE
Add quaternion degrees method

### PR DESCRIPTION
**Pull request by krishnachaitanya7**
- Add degree between quaternion method
- Add test method to calculate the degree between quaternions

### CODE REVIEW

**Review by peasant98 — APPROVED** [review]
LGTM

File: robotic_skin/calibration/utils.py [hunk 1]
```diff
@@ -395,6 +395,37 @@ def quaternion_from_two_vectors(source, target):
...
+        Quaternion a
+    :param q_2: np.ndarray
+        Quaternion b
+    :return: float
+        Angle between quaternions in degrees
+    """
+    if not (np.linalg.norm(q_1) == np.linalg.norm(q_2) == 1):
+        raise Exception("Please only pass unit quaternions")
+    angle = np.arccos(2 * ((q_1 @ q_2) ** 2) - 1)  # noqa: E999
+    angle_in_degrees = (angle / np.pi) * 180
```
Comment (watakandai) [inline robotic_skin/calibration/utils.py]: I think rad is alright.

**Review by watakandai — CHANGES_REQUESTED** [review]

File: robotic_skin/calibration/utils.py (hunk 1 above)
Comment (krishnachaitanya7) [reply robotic_skin/calibration/utils.py]: So I should be returning angle which is in radians? @watakandai

Comment (watakandai) [reply robotic_skin/calibration/utils.py]: yeah

Comment (krishnachaitanya7) [reply robotic_skin/calibration/utils.py]: Done

**Pull request merged**